packages/webapp/src/components/PopupFilter/PureCollapsibleSearch/index.jsx

```diff
@@ -44,45 +46,54 @@ export default function PureCollapsibleSearch({
 
   // Position modal searchbar
   useLayoutEffect(() => {
-    let resizeObserver;
```

Comment:
I think you can revert the changes in this component + its styles + its Storybook since you are not using it (and your changes didn't exactly address collapse).

But I'm also interested in the answer to @SayakaOno's question! Can the backdrop solution be used for finances? I am loving `<Backdrop />` -- it looks way the heck simpler!

Reply:
No I think the existing Dialog is necessary because it overlays a new search bar "as a dialog". Backdrop as I am using it just hides everything and I am lifting up the input higher than the backdrop using the zIndex and themes (saw the` (theme) => theme.zIndex + 1  ` trick in a MUI example somewhere. Hopefully we can use that to fix our other layout issues too with the menus.

This reminds me I need to make the zIndex conditional to not mess with drawer.

Reply:
Well maybe it could be used by creating a hidden input that displays conditionally...